go.mod

```diff
@@ -2,26 +2,22 @@ module github.com/balena-os/hostapp
 
 go 1.14
 
-require (
-	github.com/Microsoft/hcsshim v0.8.9 // indirect
-	github.com/containerd/containerd v1.3.7 // indirect
-	github.com/containerd/continuity v0.0.0-20200710164510-efbc4488d8fe // indirect
-	github.com/docker/distribution v2.7.1+incompatible // indirect
-	github.com/docker/docker v0.0.0-00010101000000-000000000000
-	github.com/docker/go-connections v0.4.0 // indirect
-	github.com/opencontainers/go-digest v1.0.0 // indirect
-	github.com/opencontainers/image-spec v1.0.1 // indirect
-	github.com/opencontainers/runc v0.1.1 // indirect
-	github.com/opencontainers/runtime-spec v1.0.2 // indirect
-	github.com/opencontainers/selinux v1.6.0 // indirect
-	github.com/vbatts/tar-split v0.11.1 // indirect
-	golang.org/x/sys v0.0.0-20200728102440-3e129f6d46b1
-	google.golang.org/grpc v1.31.0 // indirect
-)
+// Obtained from the vendor.conf in github.com/docker/docker v19.03.0
+replace github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc8
 
-replace (
-	// this is v19.03.12
-	github.com/docker/docker => github.com/moby/moby v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible
-	// check vendor.conf in the moby tree and take the version from there
-	github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc10
+require (
+	github.com/Microsoft/hcsshim v0.9.2 // indirect
+	github.com/containerd/continuity v0.3.0 // indirect
+	github.com/docker/distribution v2.8.1+incompatible // indirect
+	github.com/docker/docker v17.12.0-ce-rc1.0.20190604184418-5fbc0a16e22c+incompatible // v19.03.0
+	github.com/opencontainers/image-spec v1.0.2 // indirect
+	github.com/opencontainers/runc v1.1.0 // indirect
+	github.com/opencontainers/selinux v1.10.1 // indirect
+	github.com/vbatts/tar-split v0.11.2 // indirect
+	golang.org/x/net v0.0.0-20211216030914-fe4d6282115f // indirect
+	golang.org/x/sys v0.0.0-20220412211240-33da011f77ad
+	golang.org/x/text v0.3.7 // indirect
+	google.golang.org/genproto v0.0.0-20211208223120-3a66f561d7aa // indirect
+	google.golang.org/grpc v1.45.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
```

Comment:
What specific go commands did you run to adjust these dependencies, just for my own reference?

Reply:
These are automatically updated. Basically I:

* removed `go.mod`
* Initialized the project again
```
go mod init github.com/balena-os/hostapp
```
* Updated dependencies
```
go mod tidy
```
* Adjusted the docker version
* Updated dependencies again